### PR TITLE
Adding restoreTeamsCmdF function to return an error in case of a failure

### PR DESCRIPTION
**Pull request by sonalimahajan12**

### CODE REVIEW

**Comment by sonalimahajan12** [comment]
@isacikgoz Can you please assign this issue to me & review this as well.

**Comment by dsharma522** [comment]
Not sure, how there are duplicate issues for the same thing. 
I have already worked on this and opened a PR https://github.com/mattermost/mmctl/pull/581

**Comment by dsharma522** [comment]
@isacikgoz  , looks like it is duplicate issue

File: commands/team.go [hunk 1]
```diff
@@ -353,10 +355,13 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 }
 
 func restoreTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	var errs *multierror.Error
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
-			printer.PrintError("Unable to find team '" + args[i] + "'")
+			teamErr := fmt.Errorf("Unable to find team '" + args[i] + "'")
```
Comment (noxer) [inline commands/team.go:362]: You can use string interpolation here
```suggestion
			teamErr := fmt.Errorf("Unable to find team %q", args[i]")
```
using `%q` will automatically add `"` around the string.

**Review by noxer — CHANGES_REQUESTED** [review]
Hi @sonalimahajan12 and thank you for your contribution. I'd like to request one small change.

**Comment by noxer** [comment]
Duplicate of #581. Please make sure that you are assigned the original ticket first before working on it.

**Pull request closed without merging**